pkg/hubble/monitor/consumer.go

```diff
@@ -66,7 +69,7 @@ func (c *consumer) sendNumLostEvents() {
 		// We now now safely reset the counter, as at this point have
 		// successfully notified the observer about the amount of events
 		// that were lost since the previous LostEvent message
-		c.observer.GetLogger().Infof("hubble events queue is processing messages again: %d messages were lost", c.numEventsLost)
+		c.observer.GetLogger().Debugf("hubble events queue is processing messages again: %d messages were lost", c.numEventsLost)
```

Comment:
No objection on changing the level here, but can we ensure that we're always logging one "processing messages again" for each "events queue is full" message when debug is enabled?

Currently, since the latter is rate limited we can see the following pattern:
```
hubble events queue is full
processing messages again
hubble events queue is full
processing messages again
# now "hubble events queue is full" becomes rate-limited
processing messages again
processing messages again
```

Could be implemented by adding a member to the `consumer` struct to track whether we logged the hubble events queue is full message previously:

```suggestion
		if c.eventsLostLog {
			c.observer.GetLogger().Debugf("hubble events queue is processing messages again: %d messages were lost", c.numEventsLost)
			c.eventsLostLog = false
		}
```

@lambdanis what do you think?

Reply:
The "processing messages again" line contains also the number of lost events. I think it's handy to have these numbers logged at debug level (you don't need to setup a metrics collector for a quick check), but if some are logged and others not then it can become confusing (you can't even estimate the number of lost events from logs).

Maybe this line should be rephrased, to not suggest that Hubble is "back to normal" - because usually it's not, it only means that one message got in the queue, but then the queue likely is full again.

Reply:
I rephrased this log line, @kaworu do you think it's better?

Reply:
make sense, thank you!